### ISSUE TITLE
Stuck on init

### ISSUE THREAD

**Issue by kalafut** [MightyMoud/sidekick#41]
After entering my IP and email during `sidekick init` I get:

`2024/10/15 08:08:00 Failed to read known_hosts: %!s(<nil>)`

After that, subsequent invocations of `sidekick init` return:
``` 
ERROR   Sidekick config not found. Please run sidekick init.
```

Deleting `~/.config/sidekick` fixes that bit.

**Comment by kalafut** [comment]
https://github.com/MightyMoud/sidekick/blob/aa540c41674fb0a0203ec83d0e367d6b6e97515a/utils/auth.go#L98

Probably s/err/knErr

**Comment by kalafut** [comment]
Small bug here, where you should be reporting `knErr`:

https://github.com/MightyMoud/sidekick/blob/aa540c41674fb0a0203ec83d0e367d6b6e97515a/utils/auth.go#L98

I fixed that locally and saw the real error:

`/Users/kalafut/.ssh/known_hosts:5: ssh: unsupported DSA key size 2048`

Sort of annoying, since that line is some ancient host that should be irrelevant to setting up the new VPS.

**Comment by MightyMoud** [comment]
Hi @kalafut,

Thanks dude. I'll sort this error out on next release. 

Also that first error when `init` failed shouldn't happen. I'll sort that out too.

Damn man, need to set sometime aside to add tests... it's not cool like this...

**Comment by MightyMoud** [comment]
This is now fixed with latest version `0.6.6`
